drivers/hmis/app/controllers/hmis/theme_controller.rb

```diff
@@ -0,0 +1,68 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+class Hmis::ThemeController < Hmis::BaseController
+  skip_before_action :authenticate_user!
+  prepend_before_action :skip_timeout
+
+  # TODO: This is just placeholder pending other theming work, after which this should rolled in with that and be moved to the DB
+  THEMES = {
+    'christmas' => {
+      palette: {
+        primary: {
+          main: "#00aa00"
+        },
+        secondary: {
+          main: "#ff0000"
+        },
+      },
+    },
+    'halloween' => {
+      palette: {
+        mode: 'dark',
+        common: {
+          black: '#fff',
+          white: '#000'
+        },
+        primary: {
+          main: '#ff9900'
+        },
+        secondary: {
+          main: "#00aa00"
+        },
+        background: {
+          default: '#000',
+          paper: '#333'
+        }
+      },
+      components: {
+        MuiOutlinedInput: {
+          styleOverrides: {
+            root: {
+              backgroundColor: 'black',
+            },
+          },
+        },
+        MuiAppBar: {
+          styleOverrides: {
+            root: {
+              backgroundColor: 'black !important'
+            }
+          }
+        }
+      }
+    }
+  }.freeze
+
+  def index
+    which_theme = params[:ds]
+    render json: THEMES[which_theme] || {}
```

Comment:
I think this is good to merge as a first (unused) pass!

I think the Data source ID is just passed in for testing purposes, right? In production, this code should choose the theme for the data source that the request is coming from. Similar to how we `attach_data_source_id`. For example:

```
domain = URI.parse(request.origin).host
data_source_id = GrdaWarehouse::DataSource.hmis.find_by(hmis: domain).id

THEMES[data_source_id] # or however this lookup will be
```

am I missing something here @ZachCMP @eanders ?

Reply:
Yeah, that's just there as a temporary stand-in for whatever we end up using to determine theme (at least that was my understanding.)